ios/Modules/ShareMenuReactView.swift

```diff
@@ -17,7 +17,7 @@ public class ShareMenuReactView: NSObject {
     }
 
     public static func attachViewDelegate(_ delegate: ReactShareViewDelegate!) {
-        guard (ShareMenuReactView.viewDelegate == nil) else { return }
+//        guard (ShareMenuReactView.viewDelegate == nil) else { return }
```

Comment:
Does this fix the listener? 

oniOS I check initialShare on view because the listener doesn't fire